### DISCUSSION TITLE
Repo Secrets not configurable using GitHub mobile

### DISCUSSION THREAD

**Discussion by techBeck03** [community/community#125400] [Mobile]
I'd like to update a repo secret using the GitHub mobile app however I can't find any way to do so. I need a specific secret updated so I can update an auth token used in my action workflows.

**Comment by esahmed21** [comment]
Hi @techBeck03 👋

Currently, updating repository secrets through the GitHub mobile app is not supported. I'll share this feedback with our team for future enhancements. 

Thank you for bringing this to our attention!

**Reply by Abdulqaadircw** [reply]
W

**Comment by Toandz1125** [comment]
Response #2

**Comment by Mohammadseyfi1984** [comment]
> Hi @techBeck03 
> 
> Currently, updating repository secrets through the GitHub mobile app is not supported. I'll share this feedback with our team for future enhancements. 
> 
> Thank you for bringing this to our attention!
